git_machete/cmd.py

```diff
@@ -115,8 +115,37 @@ def right_arrow() -> str:
     return "->" if ascii_only else u"➔"
 
 
-def ask_if(msg: str, opt_yes_msg: Optional[str], apply_fmt: bool = True) -> str:
-    if opt_yes and opt_yes_msg:
+class CommandLineContext:
```

Comment:
I think this changeset can be merged as a self-contained PR, and other planned changes can go on separate PRs